docs/GettingStartedDocs/OP-TEE/Hardware/ScalysTrustBox.md

```diff
@@ -0,0 +1,393 @@
+will learn how to run OP-TEE's test suite as well as some of Open Enclave's
+own tests on the TrustBox.
+
+## Overview
+
+The TrustBox internally contains a Grapeboard. The latter has a MicroSD card
+reader from which the board can boot as well as a serial port offered over a
+Micro-USB Type B connector (effectively, serial-over-USB). Flashing the firmware
+requires connecting to the board over the serial connection while flashing the
+MicroSD card requires opening the TrustBox, retrieving the Grapeboard inside and
+either overwriting the MicroSD card it ships with with the root filesystem that
+you will build in this guide, or replacing the MicroSD card with your own card
+after you flash the latter with the required root filesystem.
+
+### Caution
+
+The Linux-based system that you will install to the TrustBox is configured to
+ease testing. Additionally, the secure firmware, such as OP-TEE OS, is currently
+meant for preview purposes only.
+
+> **The resulting setup is not suitable for use in production and/or hostile
+environments!**
+
+### Prerequisites
+
+- Ubuntu 18.04 LTS (64-bit)
+- 1 TrustBox
+- 1 Power supply
+- 1 MicroSD card
+  - Optional if the TrustBox you are using ships with one
+- 1 T8 Torx screwdriver
+- 1 Micro-USB Type B to USB Type A cable
+- Wired network (Ethernet)
+  - Optional
+
+**Note:** The procedure below has not been validated on Ubuntu 16.04 LTS.
+
+### Required Packages
+
+The following command installs all the packages necessary on Ubuntu 18.04 LTS:
+
+```bash
+sudo apt install -y git flex bison python-pip libssl-dev build-essential \
+    gcc-aarch64-linux-gnu g++-aarch64-linux-gnu minicom u-boot-tools     \
+    device-tree-compiler qemu-user-static udisks2
+```
+
+## Serial Communication
+
+Connect the TrustBox to your computer via its serial-over-USB port. On a
+computer running Ubuntu 18.04 LTS the TrustBox appears as a `ttyUSB#` device
+node under `/dev`.
+
+For example:
+
+```bash
+$ ll /dev/ttyUSB*
+
+crw-rw---- 1 root dialout 188, 0 Oct 21 18:14 /dev/ttyUSB0
+```
+
+To establish a duplex serial connection, replace the device node with the one on
+your system in the command below to invoke `minicom`:
+
+```bash
+sudo minicom -D /dev/ttyUSB0
+```
+
+Before the connection is usable, `minicom` must be configured to disable
+hardware flow control.
+
+Inside the `minicom` window, type:
+
+```
+<Ctrl-a> o
+```
+
+From the pop-up list, select "Serial port setup". In the new dialog, press `f`
+to switch off Hardware Flow Control. Press `<Enter>` to confirm, then `<Esc>` to
+dismiss the parent dialog. `minicom` should now be able to communicate with the
+board.
+
+Once you have finished this guide and are ready to exit `minicom`:
+
+```
+<Ctrl-a> q
+```
+
+When prompted if you would like to leave without reset, select `Yes`.
+
+**Note:** You must turn off hardware flow control every time you connect anew.
+
+## Source Code
+
+All the code necessary to build the TrustBox' firmware and software, as well as
```

Comment:
nit: "TrustBox's" (the exception of putting an apostrophe without an s to a singular noun only applies to nouns ending in s, not ones ending in x, per Chicago Manual of style :)

Reply:
:) -- Thanks!